backend/entities/Models/AccessRequestFilter.cs

```diff
@@ -0,0 +1,37 @@
+namespace Pims.Dal.Entities.Models
+{
+    public class AccessRequestFilter : PageFilter
+    {
+        /// <summary>
+        /// Get/Set - searchText
+        /// </summary>
+        public string SearchText { get; set; }
+
+        /// <summary>
+        /// Get/Set - role
+        /// </summary>
+        public string Role { get; set; }
+
+        /// <summary>
+        /// Get/Set - agency
+        /// </summary>
+        public string Agency { get; set; }
+
+        /// <summary>
+        /// Get/Set - Status
+        /// </summary>
+        public AccessRequestStatus? Status { get; set; }
+
+        public AccessRequestFilter(int page, int quantity, string[] sort, string searchText, string role, string agency,
```

Comment:
Your implementation here isn't entirely correct for best practices.  You've made the class mutable with properties that can be `set` but you haven't provided a base constructor.  It's an odd combination, even if in our use-case it most likely will never matter.

Most models should have a default base empty constructor.

Reply:
Does he need to add one in addition to the one C# adds by default to all classes?

Reply:
C# doesn't add one by default.  If someone tried to create a new instance of this class it would fail if they didn't provide any arguments.